iModelCore/BeSQLite/CloudSqlite.cpp

```diff
@@ -244,6 +244,7 @@ int CloudCache::FindToken(Utf8CP storage, Utf8CP accessName, Utf8CP containerNam
 
 /**
  * Attempt to connect this container to a CloudCache and its cloud container
+ * @note this function requires the cloudcache to be connected to the cloud container

```

Comment:
Sorry I mentioned everything that includes a cloudcache, but this one actually instantiates the cloudcache